backend/game/cards/keep_cards/victory_points_manipulation_cards/eater_of_the_dead.py

```diff
@@ -0,0 +1,11 @@
+from game.cards.keep_card import KeepCard
+from game.cards.card import Card
+from game.values import constants
+
+
+class EaterOfTheDead(KeepCard):
+    def __init__(self):
+        KeepCard.__init__(self, "Eater of the Dead", 4, "+ 3[Star] every time a monster's [Heart] goes to 0.")
```

Comment:
can just call super here

Reply:
I'd be happy to fix this, but this code came from Eater_Of_The_Dead branch. My apologize for the dup, I should probably removed it from this PR.